### PR TITLE
Add demo website

### PR DESCRIPTION
**Pull request by richardsaseun14**
image
The demo website showing on https://richardsaseun14.github.io/percircle-vue/

### CODE REVIEW

**Comment by richardsaseun14** [comment]
@teobais @miller-kevin I will be using this pull request to continue building the website if that is fine.

I am currently redoing the previous work I did before

**Comment by richardsaseun14** [comment]
I have made some more changes and they are available on https://richardsaseun14.github.io/percircle-vue/

image

File: src/components/HelloWorld.vue [hunk 1]
```diff
@@ -1,31 +1,35 @@
...
+      <h3 class="text-center">Percircle (Custom colors and reactive percents)</h3>
+
+      <ssh-pre class="mb-4" language="html" copy-button @copied="onCopiedDoSomething" dark reactive>
+        <template #copy-button>
+          <span class="small text-white">{{ copyText }}</span>
+        </template>
+        &lt;percircle :percent="{{ percent }}" :displayTextAtZero="true" progressBarColor="#1B75BC"/&gt;
+      </ssh-pre>
+
+      <!-- <pre>
```
Comment (miller-kevin) [inline src/components/HelloWorld.vue]: Clean up commented code

File: src/components/HelloWorld.vue [hunk 2]
```diff
@@ -1,31 +1,35 @@
 <template>
   <div class="container">
     <div class="mb-5">
-      <h3 class="text-center">
-        Percircle (Custom colors and reactive percents)
-      </h3>
-      <pre>
+      <h3 class="text-center">Percircle (Custom colors and reactive percents)</h3>
+
+      <ssh-pre class="mb-4" language="html" copy-button @copied="onCopiedDoSomething" dark reactive>
```
Comment (miller-kevin) [inline src/components/HelloWorld.vue]: This should be added to all of them

File: src/components/HelloWorld.vue [hunk 3]
```diff
@@ -170,6 +165,13 @@ export default {
     add(amount) {
       this.percent += amount
     },
+
+    onCopiedDoSomething() {
```
Comment (miller-kevin) [inline src/components/HelloWorld.vue]: Needs a more appropriate name

File: src/components/AppHeader.vue [hunk 4]
```diff
@@ -0,0 +1,49 @@
+<template>
+  <b-navbar toggleable="lg" type="dark" variant="info">
+    <div class="container">
+      <b-navbar-brand href="#">Percircle</b-navbar-brand>
+
+      <!-- <b-navbar-nav>
```
Comment (miller-kevin) [inline src/components/AppHeader.vue]: No leftover commented code

File: src/components/AppHeader.vue [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        <b-nav-item href="#">Link</b-nav-item>
+        <b-nav-item href="#" disabled>Disabled</b-nav-item>
+      </b-navbar-nav> -->
+
+      <!-- Right aligned nav items -->
+      <b-navbar-nav class="ms-auto">
+        <b-nav-form>
+          <!-- Place this tag where you want the button to render. -->
```
Comment (miller-kevin) [inline src/components/AppHeader.vue]: Remove tutorial comments

File: src/main.js [hunk 6]
```diff
@@ -3,6 +3,25 @@ import App from './App.vue'
...
+} from 'bootstrap-vue'
+
+// Import Bootstrap an BootstrapVue CSS files (order is important)
+import 'bootstrap/dist/css/bootstrap.css'
+import 'bootstrap-vue/dist/bootstrap-vue.css'
+
+Vue.use(LayoutPlugin)
+Vue.use(NavbarPlugin)
+Vue.use(ButtonPlugin)
+// Optionally install the BootstrapVue icon components plugin
```
Comment (miller-kevin) [inline src/main.js]: Remove tutorial comments

**Review by miller-kevin — CHANGES_REQUESTED** [review]

File: src/components/HelloWorld.vue (hunk 3 above)
Comment (richardsaseun14) [reply src/components/HelloWorld.vue]: onCopySuccess() should be fine?

Comment (miller-kevin) [reply src/components/HelloWorld.vue]: onCopy would make more sense

File: src/components/HelloWorld.vue [hunk 7]
```diff
@@ -1,153 +1,150 @@
...
-    </pre>
-    <percircle :percent="percent" progressBarColor="#1B75BC" />
-
-    <div class="buttons">
-      <button class="button" @click="add(10)">+10</button>
-      <button class="button" @click="add(-10)">-10</button>
+  <div class="container">
+    <div class="mb-5">
+      <h3 class="text-center">Percircle (Custom colors and reactive percents)</h3>
+      <ssh-pre class="mb-4" language="html" copy-button @copied="onCopySuccess" dark reactive>
```
Comment (miller-kevin) [inline src/components/HelloWorld.vue]: onCopySuccess doesn't exist

File: src/components/HelloWorld.vue [hunk 8]
```diff
@@ -1,153 +1,150 @@
...
-      </code>
-    </pre>
-    <div class="dark-background">
-      <percircle class="small dark" :percent="percent" :text="letterGrade" />
+    <div class="mb-5">
+      <h3 class="text-center">
+        Percircle (Text based, small size, dark background, uses same percent as above but translates it to a letter
+        grade)
+      </h3>
+      <ssh-pre class="mb-4" language="html" copy-button @copied="onCopySuccess" dark reactive>
```
Comment (miller-kevin) [inline src/components/HelloWorld.vue]: These all share the same copy text, if you copy one, they'll all display "copy". Should be changed, cleanest way is probably breaking each of these copy and pasted sections into a component so they each have their own copy text

**Comment by miller-kevin** [comment]
@richardsaseun14 keep in mind that in addition to the PR comments, there's also a merge conflict on package-lock.json

**Comment by richardsaseun14** [comment]
@miller-kevin i have resolved the merge conflict

**Comment by miller-kevin** [comment]
@richardsaseun14 There are still pending comments, FYI

**Comment by richardsaseun14** [comment]
@miller-kevin I have sorted out all comments.

**Comment by miller-kevin** [comment]
@richardsaseun14 There are several remaining comments

**Comment by richardsaseun14** [comment]
image

@miller-kevin Please where can I see those comment because the one available to me has been sorted. I added a screenshot to show all the comments on my end which I have resolved

**Comment by miller-kevin** [comment]
@richardsaseun14 huh, that's super weird. I still see these two which aren't visible to you

Screenshot_20220302-082236_Chrome

**Review by miller-kevin — CHANGES_REQUESTED** [review]

**Comment by richardsaseun14** [comment]
@miller-kevin i have resolved all pending comments

**Review by miller-kevin — APPROVED** [review]

**Pull request merged**